javascripts/discourse/templates/components/dc-menu.hbs

```diff
@@ -1,22 +1,10 @@
 <div class="dc-header-cloak" {{on "click" this.closeMenu}}></div>
 <div class="dc-menu-content">
   <header class="dc-menu-content-header">
-    <a
-      class="btn-transparent"
-      href={{homepageURL}}
-      {{on "click" this.closeMenu}}
-    >
-      <img
-        class="dc-menu-logo"
-        src="{{siteSettings.site_logo_url}}"
-        alt="{{siteSettings.title}}"
-      />
+    <a class="btn-transparent" href={{homepageURL}} {{on "click" this.closeMenu}}>
+      <img class="dc-menu-logo" src="{{siteSettings.site_logo_url}}" alt="{{siteSettings.title}}" />
     </a>
-    <button
-      class="btn-transparent btn-close"
-      aria-label="close menu"
-      {{on "click" this.closeMenu}}
-    >
+    <button class="btn-transparent btn-close" aria-label="close menu" {{on "click" this.closeMenu}}>
       <div class="material-icons" aria-label="close menu">
```

Comment:
The aria-label is made to describe the current element we are targeting. Duplicate the entries makes not too much sense. In this case I will rather use the aria-label only on the button node. Furthermore, it would be beneficial to remove directly the `div` within the `button`.